go/adbc/README.md

```diff
@@ -0,0 +1,251 @@
+# adbc
+
+The adbc package provides:
+
+- Go Interfaces to [Arrow Database Connectivity](https://arrow.apache.org/adbc) (ADBC) for building ADBC drivers
+- An ADBC Driver Manager package (`drivermgr`) for loading and using ADBC Drivers
```

Comment:
Hmm, the interfaces are for both using and building drivers. Maybe it's better to say something along the lines of, "Go interface definitions for using Arrow data and databases, called ADBC"